kogito-editors-js/packages/boxed-expression-component/showcase/cypress/integration/context_expression_spec.ts

```diff
@@ -15,4 +15,129 @@ describe("Context Expression Tests", () => {
+
+    // Change First context entry to return boolean
+    cy.ouiaId("expression-row-0").within(($row) => {
+      cy.ouiaId("expression-column-1").contains("ContextEntry-1").click();
+    });
+    cy.ouiaId("edit-expression-data-type").within(($container) => {
+      cy.get("input").click({ force: true });
+    });
+
+    cy.get("button:contains('boolean')").click({ force: true });
+
+    // check boolean is now also in decision table header
+    cy.ouiaType("expression-column-header-cell-info").contains("boolean").should("be.visible");
+  });
+
+  // Blocked by KOGITO-6262
+  it.skip("Define nested Decision Table", () => {
+    // Entry point for each new expression
+    cy.ouiaId("expression-container").click();
+
+    // Define new expression as Context
+    cy.ouiaId("expression-popover-menu").contains("Context").click({ force: true });
+
+    // Invoke Logic type selector for first context entry
+    cy.ouiaId("expression-row-0").within(($row) => {
+      cy.ouiaId("expression-column-2").click();
+    });
+
+    // Set first context entry as Decision Table
+    cy.ouiaId("expression-popover-menu").contains("Decision Table").click({ force: true });
+
+    // insert one output column right
+    cy.ouiaType("expression-column-header-cell-info").contains("output-1").rightclick();
+    cy.contains("Insert right").click({ force: true });
+
+    // insert one output column left
+    cy.ouiaType("expression-column-header-cell-info").contains("output-1").rightclick();
+    cy.contains("Insert left").click({ force: true });
+
+    cy.get("th:contains('output-')").should(($outputs) => {
+      expect($outputs).to.have.length(3);
+      expect($outputs.eq(1)).to.contain("output-3");
+      expect($outputs.eq(2)).to.contain("output-1");
+      expect($outputs.eq(3)).to.contain("output-2");
+    });
+  });
+
+  // Blocked by KOGITO-6262
+  it.skip("Define nested Decision Table as result", () => {
+    // Entry point for each new expression
+    cy.ouiaId("expression-container").click();
+
+    // Define new expression as Context
+    cy.ouiaId("expression-popover-menu").contains("Context").click({ force: true });
+
+    // Invoke Logic type selector for first context entry
+    cy.ouiaId("OUIA-Generated-TableRow-2").within(($row) => {
+      cy.contains("Select expression").click();
+    });
+
+    // Set first context entry as Decision Table
+    cy.ouiaId("expression-popover-menu").contains("Decision Table").click({ force: true });
+
+    // insert one output column right
+    cy.ouiaType("expression-column-header-cell-info").contains("output-1").rightclick();
+    cy.contains("Insert right").click({ force: true });
+
+    // insert one output column left
+    cy.ouiaType("expression-column-header-cell-info").contains("output-1").rightclick();
+    cy.contains("Insert left").click({ force: true });
+
+    cy.get("th:contains('output-')").should(($outputs) => {
+      expect($outputs).to.have.length(4);
```

Comment:
The length shouldn't be 3? If not, could you please explain? I'm not used to Cypress tests.

Reply:
@ljmotta good question

It is this scenario when a decision table is in the last context entry:
![Screenshot from 2021-11-29 16-55-57](https://user-images.githubusercontent.com/8044780/143900405-d5b7d8e4-5207-4d04-9e62-cabbf23a23f4.png)

```json
{"uid":"id3","logicType":"Context","name":"Expression Name","dataType":"<Undefined>","contextEntries":[{"entryInfo":{"name":"ContextEntry-1","dataType":"<Undefined>"},"entryExpression":{"name":"ContextEntry-1","dataType":"<Undefined>"},"nameAndDataTypeSynchronized":true,"id":"_E2CB8F71-B220-48FE-85D7-27CD11084613"}],"result":{"uid":"id16","logicType":"Decision Table","hitPolicy":"UNIQUE","aggregation":"","input":[{"id":"_1B55A056-BA9A-47A7-A14B-3FA556959E45","name":"input-1","dataType":"<Undefined>"}],"output":[{"id":"_36696549-A53E-4C53-B52B-2D82F509E8FC","name":"column-3","dataType":"<Undefined>","width":100},{"id":"_233021AA-6808-49D3-B3DF-9F4948C89870","name":"output-1","dataType":"<Undefined>"},{"id":"_4A41C245-9526-43D9-8CC0-EFE891912194","name":"column-2","dataType":"<Undefined>","width":100}],"annotations":[{"id":"_5C433657-D070-47B1-BC2C-89135EDA4E27","name":"annotation-1","width":100}],"rules":[{"id":"_44BC8FB6-D41F-4412-82C3-5C890213D972","inputEntries":["-"],"outputEntries":["","",""],"annotationEntries":[""]}]},"entryInfoWidth":150,"entryExpressionWidth":584,"noClearAction":false}
```

running a showcase on main, without #171

Reply:
Oh ok, now I get it.